app/livechat/server/lib/Livechat.js

```diff
@@ -156,6 +157,9 @@ export const Livechat = {
 	},
 
 	async getRoom(guest, message, roomInfo, agent, extraData) {
+		if (!this.omnichannelEnabled) {
```

Comment:
```suggestion
		if (!settings.get('Livechat_enabled')) {
```

Reply:
This same comment applies here as well -> https://github.com/RocketChat/Rocket.Chat/pull/24067/files#r776976552

Reply:
This has been applied